test-e2e/global-setup.ts

```diff
@@ -0,0 +1,12 @@
+import { type Config } from '@playwright/test'
+import { createKuboNode } from './fixtures/create-kubo-node.js'
+import { loadKuboFixtures } from './fixtures/load-kubo-fixtures.js'
+
+export default async function globalSetup (config: Config): Promise<void> {
+  const controller = await createKuboNode()
+  await Promise.all([controller.start(), loadKuboFixtures()])
+
+  process.env.KUBO_PID = `${await controller.pid()}`
```

Comment:
i thought we would need this to kill the kubo instance, but it seems like playwright handles it?